handbook/sales/index.md

```diff
@@ -114,3 +114,43 @@ Different customers and deals have different needs from our sales team. We segme
 - **Tier 1** = $100k+ ARR
 - **Tier 2** = $42k+ ARR
 - **Tier 3** = $22k+ ARR
+
+## Using HubSpot
+
+This page describes how we use HubSpot to maintain high data quality. This allows us to increase the effectiveness of all Sourcegraph teams through accurate insights. 
+
+Account Executives are responisble for maintaining HubSpot as a [source of truth](https://about.sourcegraph.com/handbook/communication#sources-of-truth). The data is synced between HubSpot and Looker every weekend so this data must be updated (at least) by end of day Friday. 
+
+### Associating Contacts to Deals
+
+It is critical to associate all of the most important contacts within a company with any new deal. This should include the technical decision-maker, the economic decision-maker (if they are different) and the original member who introduced Sourcegraph to the organization. 
+
+This allows us to evaluate the effectiveness of marketing channels and sales touchpoints that our team has with an organization. How we reached the person(s) who introduced Sourcegraph to their organization is one of the most important factors in evaluating the success of marketing activities.
```

Comment:
It also ensures that all deal-related communication is visible in the deal timeline view! Perhaps worth adding, as it's a benefit that accrues directly to the AEs who have to do this work.